titus-server-master/src/main/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManager.java

```diff
@@ -404,12 +346,7 @@ private boolean shouldRefreshScalableTargetForJob(String jobId, JobScalingConstr
     }
 
     private boolean isJobActive(String jobId) {
-        if (JobFunctions.isV2JobId(jobId)) {
-            return v2JobOperations.getJobMgr(jobId) != null && v2JobOperations.getJobMgr(jobId).isActive();
-        } else {
-            // V3
-            return v3JobOperations.getJob(jobId) != null && v3JobOperations.getJob(jobId).isPresent();
-        }
+        return v3JobOperations.getJob(jobId) != null && v3JobOperations.getJob(jobId).isPresent();
```

Comment:
The contract is to return an `Optional` so you should not need to do a null check.